src/task1-2.ts

```diff
@@ -0,0 +1,14 @@
+import { createWriteStream, createReadStream } from 'fs';
+import { pipeline } from 'stream';
+import csv from 'csvtojson';
+
+const readable = createReadStream('./hw1.csv');
```

Comment:
You have accomplished this task with the stream approach when file is not loaded fully in the RAM. If you want to get additional point you have also implement a solution when file is loaded fully into the RAM (see evaluation criteria p.3), i.e. solution should be based not on streams.